src/geometry/PolySetUtils.cc

```diff
@@ -129,4 +141,38 @@ bool is_approximately_convex(const PolySet& ps) {
 #endif
 }
 
+
+PolySet  convert_polyset(const shared_ptr<const Geometry>& geom)
+{
+  PolySet ps(3);
```

Comment:
We should ideally find a way of making a better guess at the number of vertices. ochafik@ did some work on that earlier. This kind of looks like a regression.